utils/demo/helpers/addManipulationBindings.ts

```diff
@@ -50,15 +68,7 @@ export default function addManipulationBindings(toolGroup, register = true) {
     ],
   });
   toolGroup.setToolActive(ZoomTool.toolName, {
-    bindings: [
-      {
-        mouseButton: MouseBindings.Primary, // Shift Left Click
-        modifierKey: KeyboardBindings.Shift,
-      },
-      {
-        mouseButton: MouseBindings.Secondary,
-      },
-    ],
+    bindings: zoomBindings,
```

Comment:
I'm not a fan of this being the default setting. Let's completely remove it. Instead of having an option to disable it, we should have an option to enable it for those who are interested.